examples/crate_universe_local_path/vendor_lazy_static.sh

```diff
@@ -12,6 +12,12 @@ elif [[ "$#" -eq 1 ]]; then
     path_dep_path="$1"
     copy_to="crates_from_workspace/$1"
     mkdir -p "${copy_to}"
+    sed_i=(sed -i)
+    if [[ "$(uname)" == "Darwin" ]]; then
+      sed_i=(sed -i '')
+    fi
+    "${sed_i[@]}" -e 's#manifests = \["//crates_from_workspace:Cargo\.toml"\],#manifests = ["//crates_from_workspace:Cargo.toml", "//crates_from_workspace:'"$1"'/Cargo.toml"],#g' WORKSPACE.bazel
```

Comment:
Is this to support vendoring more things?

Reply:
We run this example in two modes, and this script is for setting up the test data.

If you don't supply an arg to this script, it just vendors into a tmpdir somewhere external to the repo and pulls that in as a path dependency.

This line is to support the other mode - if you specify a relative path within this repo, it will vendor the crate into _that_ path. Now that we correctly detect that vendored crate as a member of the workspace, we need to add it to the manifests of the repo rules :)